lib/rubocop/mdsol/inject.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# The original code is from https://github.com/rubocop/rubocop-rspec/blob/master/lib/rubocop/rspec/inject.rb
```

Comment:
This file looks scary but it's also generated by https://github.com/rubocop/rubocop-extension-generator.